.github/release.yml

```diff
@@ -0,0 +1,38 @@
+changelog:
```

Comment:
That's the purpose of this file? I'd rather stick to the default ones provided by Github

Reply:
This file is to define categories with titles and related labels.

Reply:
see here https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes